src/Service/GithubApiService.php

```diff
@@ -247,17 +247,17 @@ public function cleanupRepository(ObjectEntity $repository): ?ObjectEntity
      *
      * @return ObjectEntity
      */
-    public function enrichWithOrganization(ObjectEntity $repository, array $organizationArray, Source $source): ObjectEntity
+    public function enrichWithOrganization(ObjectEntity $repository, array $repositoryArray, Source $source): ObjectEntity
     {
         $organizationSchema = $this->resourceService->getSchema('https://opencatalogi.nl/oc.organisation.schema.json', 'open-catalogi/open-catalogi-bundle');
 
-        $organizationSync = $this->syncService->findSyncBySource($source, $organizationSchema, $organizationArray['html_url']);
+        $organizationSync = $this->syncService->findSyncBySource($source, $organizationSchema, $repositoryArray['reposity']['owner']['html_url']);
```

Comment:
As far as I can see the array $repositoryArray already contains owner and owner->html_url, apart from that, there is a typo in repository.